kernel/mutex.c

```diff
@@ -130,10 +124,6 @@ int z_impl_k_mutex_lock(struct k_mutex *mutex, s32_t timeout)
 		mutex->lock_count++;
 		mutex->owner = _current;
 
-		K_DEBUG("%p took mutex %p, count: %d, orig prio: %d\n",
-			_current, mutex, mutex->lock_count,
-			mutex->owner_orig_prio);
-
```

Comment:
+1 for removing K_DEBUG, but wouldn't be good keep this info ? we could call printk directly under `ifdef CONFIG_DEBUG` guard.